packages/mc-dev-authentication/src/transformer-local.ts

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
+import path from 'path';
+import type { TCompiledHtml } from './types';
+// https://babeljs.io/blog/2017/09/11/zero-config-with-babel-macros
+import pages from /* preval */ './pages';
+
+const trimTrailingSlash = (value: string) => value.replace(/\/$/, '');
+
+// Make sure any symlinks in the project folder are resolved:
+// https://github.com/facebook/create-react-app/issues/637
+const appDirectory = fs.realpathSync(process.cwd());
+const resolveApp = (relativePath: string) =>
+  path.resolve(appDirectory, relativePath);
+const paths = {
+  appBuild: resolveApp('public'),
+};
+
+// This transformer will generate a development `login` and `logout` HTML files
+// and copy them to the application public foder.
+// This is necessary to run the application locally in production mode.
+const transformerLocal = (compiledHtml: TCompiledHtml) => {
+  const htmlLogin = pages.loginPage.replace(
+    new RegExp('__MC_API_URL__', 'g'),
```

Comment:
Will always work but still wonder if eventually we should use a very tiny templating library. I guess benefits are almost not there yet though.